husky_directory/services/search.py

```diff
@@ -61,24 +66,6 @@ def search_directory(
         duplicate_netids = set()
         timer = Timer("search_directory", context=timer_context).start()
 
-        if request_input.person_href:
-            url = base64.b64decode(request_input.person_href.encode("UTF-8")).decode(
-                "UTF-8"
-            )
-            person = self._pws.get_explicit_href(url, output_type=PersonOutput)
-            result = self.pws_translator.translate_scenario(
-                ListPersonsOutput(
-                    page_size=1, page_start=1, total_count=1, persons=[person]
-                ),
-                duplicate_netids,
-            )
-            del result["__META__"]
-            scenario = DirectoryQueryScenarioOutput(
-                populations=result, description=person.display_name
-            )
-            timer.stop(emit_log=True)
-            return SearchDirectoryOutput(scenarios=[scenario])
-
```

Comment:
All of this was replaced with the `get_listing` call below; since the individual listing now uses a different render that doesn't require the scenario hierarchy, don't need to add all this extra 💩 to make the response _look_ like a search, which is lovely.